src/store/store.ts

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import { rootReducer } from './root-reducer';
+
+export const store = configureStore({
+  reducer: rootReducer,
+});
+
+export type RootState = ReturnType<typeof store.getState>;
+export type AppDispatch = typeof store.dispatch;
```

Comment:
Let's export from this file only store and move these types to types/app
-- app-dispatch.ts
-- root-state.ts
-- index.ts